h5netcdf/attrs.py

```diff
@@ -22,8 +22,14 @@ def __init__(self, h5attrs, check_dtype):
         self._check_dtype = check_dtype
 
     def __getitem__(self, key):
+        import h5py
+
         if key in _HIDDEN_ATTRS:
             raise KeyError(key)
+        if isinstance(self._h5attrs[key], h5py.Empty):
```

Comment:
Maybe add a comment referencing the issue where I explained what is going on down?

Reply:
Done!